lldb/source/Commands/CommandOptionsProcessLaunch.cpp

```diff
@@ -38,7 +41,13 @@ Status CommandOptionsProcessLaunch::SetOptionValue(
   case 's': // Stop at program entry point
     launch_info.GetFlags().Set(eLaunchFlagStopAtEntry);
     break;
-
+  case 'm': // Stop at user entry point
+  {
+    TargetSP target_sp =
+        execution_context ? execution_context->GetTargetSP() : TargetSP();
```

Comment:
There's another option below (`'a'`) that also needs the `target_sp`. Let's hoist this out of the switch and reuse it in all the options.